openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -2,33 +2,31 @@
 {% load i18n %}
 <div class="assessment__fields">
     <ol class="list list--fields assessment__rubric">
-
         {% for criterion in rubric_criteria %}
         <li
-            class="field field--radio is--required assessment__rubric__question is--showing ui-slidable__container {% if criterion.options %}has--options{% endif %}"
-            id="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}"
+            class="field field--radio is--required assessment__rubric__question is--showing ui-slidable__container {% if criterion.options %}has--options{% endif %} {{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}"
         >
             <h4 class="question__title ui-slidable__control">
-                <button class="ui-slidable" aria-expanded="true" id="oa_rubric_{{ submission.uuid }}" aria-controls="oa_rubric_{{ submission.uuid }}_content">
+                <button class="ui-slidable" aria-expanded="true" id="oa_rubric__{{ rubric_type }}__{{ submission.uuid }}__{{ criterion.order_num }}" aria-controls="oa_rubric__{{ rubric_type }}__{{ submission.uuid }}__{{ criterion.order_num }}__content">
                     <span class="icon fa fa-caret-right" aria-hidden="true"></span>
                     <span id="{{ rubric_type }}__assessment__rubric__prompt--{{ criterion.order_num }}" class="ui-slidable__control__copy question__title__copy">{{ criterion.prompt }}</span>
                     <span class="label--required sr">* ({% trans "Required" %})</span>
                 </button>
             </h4>
 
-            <div class="ui-slidable__content" aria-labelledby="oa_rubric_{{ submission.uuid }}" id="oa_rubric_{{ submission.uuid }}_content">
+            <div class="ui-slidable__content" aria-labelledby="oa_rubric__{{ rubric_type }}__{{ submission.uuid }}__{{ criterion.order_num }}" id="oa_rubric__{{ rubric_type }}__{{ submission.uuid }}__{{ criterion.order_num }}__content">
                 <div class="question__answers">
                     <div role="group" aria-labelledby="{{ rubric_type }}__assessment__rubric__prompt--{{ criterion.order_num }}">
                         {% for option in criterion.options %}
                         <div class="answer">
                             <div class="wrapper--input">
                                 <input type="radio"
                                        data-criterion-name="{{ criterion.name }}"
-                                       id="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}"
-                                       class="answer__value"
+                                       class="answer__value rubric_{{ criterion.order_num }}_{{ option.order_num }}"
```

Comment:
Is this class value rubric_{{ criterion.order_num }}_{{ option.order_num }} actually needed (since there is also still an ID on the element)?